app/controllers/api/v1/users_controller.rb

```diff
@@ -3,21 +3,24 @@ class Api::V1::UsersController < ApplicationController
 
   def create
     user = User.new(user_params)
+    
     if !user.email.include?('@')
-      render json: {error: "Error creating user. A valid email must be provided."}, status: 400
+      render json: {error: "Error creating subscriber. A valid email must be provided."}, status: 400
     elsif user.save
       UserMailer.registration(user).deliver_now
-      render json: {success: "User created. A confirmation email has been sent to #{user.email}."}, status: 200
+      render json: {success: "You are now registered to receive notifications about upcoming elections in your state. A confirmation email has been sent to #{user.email}."}, status: 200
+    elsif User.find_by(email: user.email).present?
+      render json: {error: "This email is already susbscribed to receive election notifications."}, status: 400
```

Comment:
@larroyo1 @AlexMMcConnell could we please adjust this response to `{error: "This #{user.email} is already susbscribed to receive election notifications."}`

Our form will have already cleared all of the inputs, and maybe our user doesn't remember which email they had attempted to input.

Thanks!